Description: Clear the path of config_file /usr/bin/opencc
 Give out the path of config_file: /usr/lib/${DEB_HOST_MULTIARCH}/opencc/*.ini
Author: Kan-Ru Chen <koster@debian.org>, YunQiang Su <wzssyqa@gmail.com>

Index: opencc/doc/opencc.1
===================================================================
--- opencc.orig/doc/opencc.1	2012-03-29 04:35:59.000000000 +0800
+++ opencc/doc/opencc.1	2012-03-29 04:38:50.720283041 +0800
@@ -16,6 +16,11 @@
 \fB\-c\fR
 Load dictionary configuration from config_file.
 .IP
+Package libopencc1 has some configuration file preset: /usr/lib/${DEB_HOST_MULTIARCH}/opencc/*.ini;
+the ${DEB_HOST_MULTIARCH} is something like "x86_64-linux-gnu" (AMD64)
+or "i386-linux-gnu" (i386), or something else;
+you can get it by `dpkg-architecture \-qDEB_HOST_MULTIARCH`.
+.IP
 Note:
 .IP
 Text from standard input will be read if input_file is not set and will be written to standard output if output_file is not set.
